### ISSUE TITLE
Can't execute maven related commands using ps

### ISSUE THREAD

**Issue by bsaby** [Eskibear/vscode-maven#39]
1. Specify maven executable path in settings
"maven.executable.path": "C:\\Users\\v-sunxi\\Downloads\\apache-maven-3.5.2\\bin\\mvn"
2. Can't execute maven related commands using ps
![image](https://user-images.githubusercontent.com/22470312/34135842-fea062c4-e49d-11e7-9a1d-8ea74a2f709e.png)

**Comment by Eskibear** [comment]
Fixed in #40 .
BTW, you'd better use `mvn.cmd` in Windows.
```
"maven.executable.path": "C:\Users\v-sunxi\Downloads\apache-maven-3.5.2\bin\mvn.cmd"
```

**Comment by bsaby** [comment]
Test passed. 
Build used: build20
